src/js/common/helpers/login-helpers.js

```diff
@@ -27,6 +27,7 @@ export function getUserData(dispatch) {
       if (userData.first_name) {
         sessionStorage.setItem('userFirstName', userData.first_name);
       }
+      dispatch(updateProfileField('savedForms', json.data.attributes.in_progress_forms));
```

Comment:
TBH, I intentionally ignored everything after `handleVerify` when I touched this file. This part looked scary and made me think our profile code could use some refactoring.